doc/whats_new.rst

```diff
@@ -1,14 +1,60 @@
-0.1.5
-=======
+0.2
+===
 
 Changelog
 ---------
 
 The new minimum required version of scikit-learn is 0.13
 
-- Fix to control the behaviour of cut_coords or number of cuts in plot_stat_map.
-  For consistency, number of cuts is changed to default value 7.
-
+New features
+............
+   - The new module named as connectome.
+   - The new class ConnectivityMeasure in connectome module has tools for
```

Comment:
Can you use the sphinx markup to have links to the corresponding class: :class:`nilearn.connectome.ConnectivityMeasure`